### DISCUSSION TITLE
Winmerge does bad job of handling python indents

### DISCUSSION THREAD

**Discussion by cjthedj** [WinMerge/winmerge#1291] [Q&A]
Hi 
  If I modify a python program to add an extra level of "if"  everything underneath has to indent.
With whitespace set to "compare", an indent completely confuses the comparator and all the - otherwise identical  - text fails to compare.
However with whitespace set to "ignore" al the text matches but I cannot then see any faulty indents in the rest of the file.
  Is there some way of achieving "show indent changes" and "do not lose the plot on indented changes" at the same time?
Its a pain having to run the comparator twice with different setting to do a python file compare.
  Others have asked the same question elsewhere. For example:-

https://softwarerecs.stackexchange.com/questions/19538/is-there-a-diff-tool-that-handles-python-indentation-changes

**Comment by sdottaka** [comment]
It doesn't always work, but in this example, enabling the "Match similar lines" option in the Compare / General category of the Options dialog will give you the results you expect.

![image](https://user-images.githubusercontent.com/98126/162442924-2405fec6-54a4-496c-8874-620ce1a89739.png)
